### DISCUSSION TITLE
How can I get the output at the head of the output panel buffer?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [michaelb/sniprun#256] [Q&A] [answered]
Right now, it always append to the bottom of output panel buffer.

I need to switch to that buffer window then press `G` to navigate down to the bottom to see the new output of the run.

So is it possible to append the new output to the beginning of the output panel?

**Comment by michaelb** [comment, marked as answer]
hmmm, that should not happen (if we're talking about the display mode Terminal and not something else). Sniprun sets a `scrollback` to the terminal buffer that take its default from vim.o.scrollback (almost always -1)

You may want to execute `:lua print(vim.o.scrollback)` to check whether your config changed this.

You may change this by setting the scrollback option to -1 only for sniprun: https://michaelb.github.io/sniprun/sources/display_modes.html#terminal

**Reply by LintaoAmons** [reply]
I open a new project found it's working, the Terminal will scroll down.

OK, seems like it's a single case~

Thanks for your reply~
